kubernetes_secrets.yaml.template

```diff
@@ -0,0 +1,51 @@
+Templates for the secretes required by kubernetes to run the app. 
+These are based on providing all values as plain text. To use base64 encoded values use 'data' property instead of 'stringData'.
+
+----------------------------------
```

Comment:
Minor but if you commented these top two lines and reduced the breaks to exactly three hyphens, then this file could be applied directly as a single file rather than needing to split it out into multiple.

Reply:
Oooh, I tried to find out how to combine them in a single file and couldn't find anything, that's very helpful!